src/python/Utils/PortForward.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+"""
+_PortForward_
+
+A decorator for swapping ports in an url
+"""
+from __future__ import print_function
```

Comment:
It isn't clear to me whether we also need these 2 future imports, do we?

Reply:
it was basically because pylint was complaining about python3 incompatibility, and it still does for the missing `division` import from `future`

Reply:
Then I'd suggest to replace `print_function` by `division`.

And answering my own question, `standard_library` is required for importing `urlparse` compatible with python2 and python3.

Reply:
You are absolutely correct about `urlparse`. This took me some time to figure it out myself :)

Reply:
> Then I'd suggest to replace print_function by division

pylint complains if any of those is missing, So I'd better add them both

Reply:
Done